src/test/java/com/google/javascript/clutz/OptionsTest.java

```diff
@@ -104,11 +104,11 @@ public void testExternsShouldBeUnique() throws Exception {
         new Options(
             new String[] {
               "--externs",
-              "javascript/common/dom.js",
+              "my/root/extern.js",
               "--depgraphs",
               DepgraphTest.DEPGRAPH_PATH.toFile().toString(),
```

Comment:
Add a comment saying that depgraph has my/root/extern.js and my/nonroot/extern.js